static/app/views/settings/organizationMembers/inviteMember/orgRoleSelect.tsx

```diff
@@ -54,7 +54,7 @@ class OrganizationRoleSelect extends Component<Props> {
                 <Label>
                   <Radio id={id} value={name} checked={id === roleSelected} readOnly />
                   <div style={{flex: 1, padding: '0 16px'}}>
-                    {name} {isRetired && t('(Deprecated)')}
```

Comment:
`Deprecated` will be added from the string returned from the backend in PR #39868